packages/components/src/Drawer/Drawer.component.js

```diff
@@ -164,7 +164,10 @@ function combinedFooterActions(onCancelAction, footerActions) {
 	}
 	const enhancedFooterActions = Object.assign({}, footerActions);
 	if (footerActions && footerActions.actions && footerActions.actions.left) {
-		enhancedFooterActions.actions.left.push(onCancelAction);
+		enhancedFooterActions.actions = {
+			...enhancedFooterActions.actions,
+			left: [...enhancedFooterActions.actions.left, onCancelAction],
+		};
```

Comment:
Cant we use lodash for the checking: 
if (_lodash.get(footerActions, 'actions') {
enhancedFooterActions.actions = {
...enhancedFooterAtions.actions, left: [ get(enhancedFooterActions, 'actions.left'), cancelAction]
}

Reply:
done